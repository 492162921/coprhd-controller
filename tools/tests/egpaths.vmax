EG:  16host  EM:  all
Existing volumes:  StringMap {60000970000197000143533031333246=1}
Hostname ... Initiator WWN ... Port Name ... Zone Name ... Count
eg 16hosts em 16hosts_143 ssys urn:storageos:StorageSystem:c9c20590-55b5-4fd1-8e43-3929b8aa2bac:vdc1
host16a.org 10:50:57:57:57:57:57:02 a973-4154  f06b-4035 FA-2D:9 SDS_host16aorg_575757575702_0143_FA2D9 1
host16a.org 10:50:57:57:57:57:57:03 535b-4955  601e-4648 FA-2D:6 SDS_host16aorg_575757575703_0143_FA2D6 1
host16b.org 10:50:57:57:57:57:57:04 a460-4937  f06b-4035 FA-2D:9 SDS_host16borg_575757575704_0143_FA2D9 1
host16b.org 10:50:57:57:57:57:57:06 d81f-4516  601e-4648 FA-2D:6 SDS_host16borg_575757575706_0143_FA2D6 1
host16c.org 10:50:57:57:57:57:57:09 3947-4b1c  f06b-4035 FA-2D:9 SDS_host16corg_575757575709_0143_FA2D9 1
host16c.org 10:50:57:57:57:57:57:0A 8068-48bd  601e-4648 FA-2D:6 SDS_host16corg_57575757570A_0143_FA2D6 1
host16d.org 10:50:57:57:57:57:57:0C 2ce0-49bf  601e-4648 FA-2D:6 SDS_host16dorg_57575757570C_0143_FA2D6 1
host16d.org 10:50:57:57:57:57:57:0F 5324-481f  f06b-4035 FA-2D:9 SDS_host16dorg_57575757570F_0143_FA2D9 1
host16e.org 10:50:57:57:57:57:57:11 8a2f-46b9  f06b-4035 FA-2D:9 SDS_host16eorg_575757575711_0143_FA2D9 1
host16e.org 10:50:57:57:57:57:57:13 8730-400f  601e-4648 FA-2D:6 SDS_host16eorg_575757575713_0143_FA2D6 1
host16f.org 10:50:57:57:57:57:57:14 f0ec-4e24  601e-4648 FA-2D:6 SDS_host16forg_575757575714_0143_FA2D6 1
host16f.org 10:50:57:57:57:57:57:15 7a7c-46ef  f06b-4035 FA-2D:9 SDS_host16forg_575757575715_0143_FA2D9 1
host16g.org 10:50:57:57:57:57:57:18 24e6-4fd2  f06b-4035 FA-2D:9 SDS_host16gorg_575757575718_0143_FA2D9 1
host16g.org 10:50:57:57:57:57:57:19 acd6-49cc  601e-4648 FA-2D:6 SDS_host16gorg_575757575719_0143_FA2D6 1
host16h.org 10:50:57:57:57:57:57:1D 019e-466e  601e-4648 FA-2D:6 SDS_host16horg_57575757571D_0143_FA2D6 1
host16h.org 10:50:57:57:57:57:57:1E 6c00-499a  f06b-4035 FA-2D:9 SDS_host16horg_57575757571E_0143_FA2D9 1
host16i.org 10:50:57:57:57:57:57:22 03e1-41fa  f06b-4035 FA-2D:9 SDS_host16iorg_575757575722_0143_FA2D9 1
host16i.org 10:50:57:57:57:57:57:23 3f21-43d8  601e-4648 FA-2D:6 SDS_host16iorg_575757575723_0143_FA2D6 1
host16j.org 10:50:57:57:57:57:57:25 974f-48ea  f06b-4035 FA-2D:9 SDS_host16jorg_575757575725_0143_FA2D9 1
host16j.org 10:50:57:57:57:57:57:27 a86a-436e  601e-4648 FA-2D:6 SDS_host16jorg_575757575727_0143_FA2D6 1
host16k.org 10:50:57:57:57:57:57:28 7280-4724  f06b-4035 FA-2D:9 SDS_host16korg_575757575728_0143_FA2D9 1
host16k.org 10:50:57:57:57:57:57:2B cc1b-44a4  601e-4648 FA-2D:6 SDS_host16korg_57575757572B_0143_FA2D6 1
host16l.org 10:50:57:57:57:57:57:2E 5298-4761  601e-4648 FA-2D:6 SDS_host16lorg_57575757572E_0143_FA2D6 1
host16l.org 10:50:57:57:57:57:57:2F 4437-4b7f  f06b-4035 FA-2D:9 SDS_host16lorg_57575757572F_0143_FA2D9 1
host16m.org 10:50:57:57:57:57:57:30 f4d7-4194  601e-4648 FA-2D:6 SDS_host16morg_575757575730_0143_FA2D6 1
host16m.org 10:50:57:57:57:57:57:33 e8b0-43af  f06b-4035 FA-2D:9 SDS_host16morg_575757575733_0143_FA2D9 1
host16n.org 10:50:57:57:57:57:57:34 f4b2-4997  f06b-4035 FA-2D:9 SDS_host16norg_575757575734_0143_FA2D9 1
host16n.org 10:50:57:57:57:57:57:37 2b33-46a7  601e-4648 FA-2D:6 SDS_host16norg_575757575737_0143_FA2D6 1
host16o.org 10:50:57:57:57:57:57:38 8c79-421c  f06b-4035 FA-2D:9 SDS_host16oorg_575757575738_0143_FA2D9 1
host16o.org 10:50:57:57:57:57:57:3A 7cdd-438d  601e-4648 FA-2D:6 SDS_host16oorg_57575757573A_0143_FA2D6 1
host16p.org 10:50:57:57:57:57:57:3E f077-48fc  f06b-4035 FA-2D:9 SDS_host16porg_57575757573E_0143_FA2D9 1
host16p.org 10:50:57:57:57:57:57:3F 1d70-4404  601e-4648 FA-2D:6 SDS_host16porg_57575757573F_0143_FA2D6 1
