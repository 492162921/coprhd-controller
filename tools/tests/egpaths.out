EG:  h1025a  EM:  all
Hostname ... Initiator WWN ... Port Name ... Zone Name ... Count
eg h1025a em V2_aclus025A_h1025aorg_301 ssys urn:storageos:StorageSystem:91921229-b289-46a1-8584-fdbfa04e08e1:vdc1
h1025a.org 10:00:50:56:A2:02:5A:12 8eb2-4ee0  4f25-4c48 B0-FC01 SDS_h1025aorg_5056A2025A12_0301_21B0FC01 1
eg h1025a em V1_aclus025A_h1025aorg_300 ssys urn:storageos:StorageSystem:91921229-b289-46a1-8584-fdbfa04e08e1:vdc1
Existing volumes:  StringMap {60000000000000099931603130200002=4, 60000000000000099900041419700003=3}            <<<<<<<<<<<<<<<<<<<<< WRONG
h1025a.org 10:00:50:56:A2:02:5A:11 137f-4645  bb8f-44ef B0-FC02 SDS_h1025aorg_5056A2025A11_0300_11B0FC02 2
h1025a.org 10:00:50:56:A2:02:5A:10 85c5-455d  2174-4cdc A0-FC03 SDS_h1025aorg_5056A2025A10_0300_11A0FC03 2
